packages/main/src/TableRow.ts

```diff
@@ -79,6 +79,15 @@ class TableRow extends TableRowBase {
 	@property({ type: Boolean })
 	navigated = false;
 
+	/**
+	 * Defines whether the row is movable.
+	 *
+	 * @default false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	movable = false;
```

Comment:
technically not required, as you could just simply set `draggable="true"` directly on the `ui5-table-row` as it is in the light DOM.

But I guess, this would make sense for consistency across the API

Reply:
Yes, please use `movable` for consistency. It is also more meaningful in conjunction with the:

- `move` and `move-over` events
- potential keyboard support from your side